.github/workflows/tests.yml

```diff
@@ -41,83 +42,85 @@ jobs:
         key: pip-${{ matrix.python-version }}-${{ hashFiles('requirements-*.txt') }}
         restore-keys: |
           pip-${{ matrix.python-version }}-
-  tests_7_17:
+  tests_8_0:
     needs: [install_es, install_py]
     runs-on: ubuntu-latest
     strategy:
       fail-fast: true
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10", pypy-3.8]
+#        python-version: [3.7, 3.8, 3.9, "3.10", pypy-3.8]
+        python-version: ["3.10"]
     env:
       OS: ubuntu-latest
       PYTHON: ${{ matrix.python-version }}
+      ES_JAVA_OPTS: "-Xms256m -Xmx512m"
```

Comment:
Did you already try decreasing these values even further?

Reply:
No, there seems to be plenty of available:
https://github.com/ClearcodeHQ/pytest-elasticsearch/runs/6580001952?check_suite_focus=true#step:3:198